src/scripts/main.js

```diff
@@ -1 +1,46 @@
 'use strict';
+
+function newMessage(messages, classElement) {
+  const div = document.createElement('div');
+
+  div.innerHTML = messages;
+  div.classList.add(classElement);
+  div.setAttribute('data-qa', 'notification');
+  document.body.append(div);
+}
+
+new Promise((resolve, reject) => {
+  document.addEventListener('mousedown', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(reject, 3000, 'First promise was rejected');
+}).then((message) => newMessage(message, 'success'))
+  .catch((message) => newMessage(message, 'warning'));
+
+new Promise((resolve, reject) => {
+  document.addEventListener('mousedown', ev => {
+    if ((ev.button === 0) || (ev.button === 2)) {
+      resolve('Second promise was resolved');
+    }
+  });
+}).then((message) => newMessage(message, 'success'));
+
+new Promise((resolve, reject) => {
+  let buttonLeft = false;
+  let buttonRight = false;
+
+  document.addEventListener('mouseup', ev => {
```

Comment:
```suggestion
  document.addEventListener('mousedown', ev => {
```
Думаю, тут краще буде mousedown, тому що зараз третій проміс спрацьовує не зовсім так, як треба

Також було б добре додати на івент contextmenu preventDefault, щоб не з'являлось контекстне меню

Reply:
Спасибо! Исправил!